site/components/cart/CartItem/CartItem.tsx

```diff
@@ -119,9 +120,14 @@ const CartItem = ({ item, currencyCode }: ICartItem) => {
 										<span
 											className='mx-2 rounded-full bg-transparent border w-5 h-5 p-1 text-accent-9 inline-flex items-center justify-center overflow-hidden'
 											style={{
-												backgroundColor: `${option.value}`,
+												background:
+													colorMap[
+														option.value
```

Comment:
what happens if the value is not in the map? can you pls ensure it doesnt crash and also give a default color

Reply:
Done!